### ISSUE TITLE
[BUG] Disable root certificates auto update will cause problems

### ISSUE THREAD

**Issue by wevsty** [meetrevision/playbook#15]
I found that using python urllib in ReviOS kind of pops up an exception directly.

Example:
python script
```python
import urllib.request

link = 'https://letsencrypt.org/'
with urllib.request.urlopen(link) as response:
    source = response.read()
    print(source)
```

output error:
```python
E:\>E:\debug.py
Traceback (most recent call last):
  File "C:\USER_PROGRAMS\Python\lib\urllib\request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 1283, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 1329, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 1278, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 1038, in _send_output
    self.send(msg)
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 976, in send
    self.connect()
  File "C:\USER_PROGRAMS\Python\lib\http\client.py", line 1455, in connect
    self.sock = self._context.wrap_socket(self.sock,
  File "C:\USER_PROGRAMS\Python\lib\ssl.py", line 513, in wrap_socket
    return self.sslsocket_class._create(
  File "C:\USER_PROGRAMS\Python\lib\ssl.py", line 1071, in _create
    self.do_handshake()
  File "C:\USER_PROGRAMS\Python\lib\ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)
```

So I researched why the results.
I noticed that playbook kind of setting root certificates updates.
Although the certificate update feature is enabled, it is not clear why automatic updates do not in fact take place.
https://github.com/meetrevision/playbook/blob/910d561b9db625bef50985428d7b35359c73c84a/src/Configuration/features/revision/registry/configure-privacy.yml#L209

In a clean install of Windows, Windows only have a few trusted root certificates are installed by default, Some root certificates like letsencrypt are not installed by default.
This means that if root certificate update is failed, accessing certificates issued by non-default installed root certificates using a range of API for system-trusted root certificates will result in errors.

I think this is a bug.
There are two ways to solve the problem:
1, we can choose to allow Windows to automatically update the root certificate in the background, and if the automatic update is successful this problem will be solved.
2, perhaps we can provide the option to update the root certificate manually (for example, in Revision Tool)

For users who already have the current version installed, I wrote a simple powershell script to solve this problem.
```powershell
# Create temporary file path
$TempFile = (New-TemporaryFile).FullName

# Download latest SST from Microsoft
CertUtil -generateSSTFromWU -f $TempFile

# If the download is successful, the file size should be larger than 0
if ( (Get-Item $TempFile | Measure-Object -Property Length -Sum).sum -gt 0 )
{
    $SST_File = Get-ChildItem -Path $TempFile
    $SST_File | Import-Certificate -CertStoreLocation Cert:\LocalMachine\Root\
    $SST_File | Import-Certificate -CertStoreLocation Cert:\LocalMachine\AuthRoot\
    Write-Output "Update root certificate successfully."
}
else
{
    Write-Output "Failed to update root certificate."
}

# Delete temporary files
Remove-Item -Path $TempFile
```

**Comment by dreammjow** [comment]
I think It's more appropriate to let Windows automatically update certificates

**Comment by wevsty** [comment]
@dreammjow 
You're right, setting ```DisableRootAutoUpdate``` to 0 means turning on automatic updates.
But I don't know why, in essence the system won't try to update automatically.
Maybe it has something to do with Windows Update Services, it seems that automatic updates will return to normal after manually executing ```certutil``` updates, you can also try testing to see the result.

**Comment by wevsty** [comment]
I tried to update my laptop's BIOS today, but the updater prompted me that it couldn't load the driver, so I looked into the problem again.

I found out that just adding the certificate to ```Cert:\LocalMachine\Root\``` is not enough, it also needs to be added to ```Cert:\LocalMachine\AuthRoot\``` otherwise part of the driver will not load because the digital signature cannot be verified.

The script in the post has been updated to address this point.
